doc/monitoring/metrics_reference.rst

```diff
@@ -993,3 +993,40 @@ Read view statistics
 
         *   -   ``tnt_memtx_index_read_view``
             -   Memory (in bytes) held for read views.
+
+
+~~~~~~~~~~~~~~~~~~~~~~~
+Tarantool configuration
+~~~~~~~~~~~~~~~~~~~~~~~
+
+These metrics are available starting from Tarantool 3.0.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_config_alerts``
+            -   Count of current instance :ref:`configuration apply alerts <_config_api_reference_info>`.
+                ``{level="warn"}`` label covers warnings and
+                ``{level="error"}`` covers errors.
+
+        *   -   ``tnt_config_status``
+            -   The status of current instance :ref:`configuration apply <_config_api_reference_info>`.
```

Comment:
These are for Sphinx documentation build, they don't render on Github (and will render properly only as a part of tarantool/doc since they reference a base documentation path). On the other hand, you're still right and they were broken. Now everything is fine: 

![image](https://github.com/tarantool/metrics/assets/20455996/9f51a35a-6a20-4448-8c7b-eaa89713aa0f)
(you can check here with proper credentials: https://docs.d.tarantool.io/en/doc/metrics-1.2.0/book/monitoring/metrics_reference/)